specs/specTemplate/sampleComponentSpec.md

```diff
@@ -109,14 +167,31 @@ Detailed description of how input behaves. Logic, corner cases, patterns, etc.
 
 
 
-### Touch
+#### **Touch**
 
 | Event | Action              | NOTE                    |
 | ----- | ------------------- | ----------------------- |
 | tap   | what happens on tap | Side note (if relevant) |
 | drag  |                     |                         |
 
 
+
+## RTL
+
+> We are deciding on how are we going to handle the RTL. Detailed description will be added later.
+
+
+
+## DOM structure
+
+Each component should have a visual style guide for all of its visual states and elements structure. This style guide will be based on the **Style API**, and a visual theme agreed upon as our reset style (Wix style).
+
```

Comment:
maybe we should soften this then a little. i do want more details than we're putting in today.